### ISSUE TITLE
V8: Quirky situation when limiting a user or user-group to a start node thats not placed in the root.

### ISSUE THREAD

**Issue by nielslyngsoe** [umbraco/Umbraco-CMS#4696]
When setting the start-node for a user or user-group that is not located in the root the visual experience becomes funny.

The start node node appears as a root-node, but besides it you will see the real tree and within it the allowed start-node will appear.

![image](https://user-images.githubusercontent.com/6791648/53170657-f1f7b500-35e0-11e9-8dde-5cdce84f190f.png)

**Comment by Shazwazza** [comment]
That one is a little odd, must be a bug that has crept in, there's a ton of logic in c# to deal with these things and there's a bunch of different factors that are taken into account. We normally show the whole tree structure so that the user knows where they are especially if they have multiple start nodes selected.

**Comment by mvanhelmont** [comment]
The problem that @nielslyngsoe describes also applies to media. Pull request fixes both, but after testing the pull request i also found another bug that's only applies to media.

You get Create an item under Folder 1 with title Reload, but that's not the startnode.

![media](https://user-images.githubusercontent.com/5310629/57330248-c1baaf80-7115-11e9-8495-77514f520570.png)

**Comment by mvanhelmont** [comment]
I have added a new commit to fix this, was just placing the next line on the right place.

```
//set the default
menu.DefaultMenuAlias = ActionNew.ActionAlias;
```

**Comment by inetzo** [comment]
Hi Marcel, thank you very much for creating this fix! This issue is still relevant and does not seem to be fixed in 8.2.1. Hopefully this will be fixed soon, setting startnodes is one of the main features of Umbraco in my opinion.

**Comment by kjac** [comment]
Fixed in #6815
